__init__.py

```diff
@@ -314,6 +331,9 @@ def register():
 	bpy.types.Scene.exportHiddenGeometry = bpy.props.BoolProperty(name = "Export Hidden Geometry", default = True)
 	bpy.types.Scene.blenderF3DScale = bpy.props.FloatProperty(name = "F3D Blender Scale", default = 100)
 
+	bpy.types.Scene.fast64 = bpy.props.PointerProperty(type=Fast64_Properties, name="Fast64 Properties")
+	bpy.app.handlers.load_post.append(after_load)
```

Comment:
it's probably a good idea to remove the handler in unregister() too

Reply:
done!